sgv2-filter.user.js

```diff
@@ -410,9 +448,11 @@ function createFilterUiFilterOptionsRow() {
   minLevelToDisplayInput.onchange = function() {
     // Filter out invalid values
     var minLevelToDisplayInputValue = parseInt(minLevelToDisplayInput.value);
-    if (minLevelToDisplayInputValue < 0 || minLevelToDisplayInputValue > 10 || minLevelToDisplayInputValue > maxLevelToDisplay) {
-      minLevelToDisplayInput.value = minLevelToDisplay;
-    } else if (minLevelToDisplay != minLevelToDisplayInputValue) {
+    if (minLevelToDisplayInputValue < DEFAULT_MIN_LEVEL_TO_DISPLAY) {
```

Comment:
I don't think it works now :) You set the desired value to the minLevelToDisplayInput.value, but you don't save i to the minLevelToDisplayInput and to the GM_setValue. I think you need to separate the last "else if" and make it just if. The two "checking" conditions should then save it not only to the minLevelToDisplayInput.value, but to the minLevelToDisplayInputValue as well.